test/src/org/labkey/test/tests/response/ConfigAndEnrollTest.java

```diff
@@ -75,57 +74,44 @@ public void testStudyName()
         _containerHelper.createProject(PROJECT_NAME01, FOLDER_TYPE);
         goToProjectHome(PROJECT_NAME01);
 
-        setupPage = new SetupPage(this);
-
-        log("Validate the prompt.");
-        assertEquals("The prompt is not as expected.", PROMPT_NOT_ASSIGNED, setupPage.getStudySetupWebPart().getPrompt());
-        setupPage.validateSubmitButtonDisabled();
+        ForwardingTab forwardingTab = ForwardingTab.beginAt(this);
+        forwardingTab.validateSaveButtonDisabled();
 
         log("Set a study name.");
-        setupPage.getStudySetupWebPart().setShortName(STUDY_NAME01);
-        setupPage.validateSubmitButtonEnabled();
-        setupPage.getStudySetupWebPart().clickSubmit();
-
-        log("Validate that the submit button is disabled after you click it.");
-        assertFalse("Submit button is showing as enabled, it should not be.", setupPage.getStudySetupWebPart().isSubmitEnabled());
+        forwardingTab.setInputId(STUDY_NAME01);
+        forwardingTab.validateSaveButtonEnabled();
+        forwardingTab.saveAndExpectSuccess();
 
-        log("Remove the web part,bring it back and validate the study name is still there.");
-        setupPage.getStudySetupWebPart().remove();
```

Comment:
We have removed the web part in favor of adding the UI to the MyStudies Response Server  Folder Management tab.